tests/core/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+cmake_policy(SET CMP0057 NEW)
+include(GoogleTest)
```

Comment:
This doesn't exist in master.

Reply:
This will force us to use cmake 3.10. Droped.

Reply:
`s/master/CMake < 3.10/`